### PR TITLE
fix in 23.6, example of bad code is not enough bad

### PR DESCRIPTION
**Pull request by RslnMtvv**
Code corrected according to the comment of example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2893,13 +2893,13 @@ Other Style Guides
     // in some other file
     // bad
     import CheckBox from './checkBox'; // PascalCase import/export, camelCase filename
-    import FortyTwo from './FortyTwo'; // PascalCase import/filename, camelCase export
-    import InsideDirectory from './InsideDirectory'; // PascalCase import/filename, camelCase export
+    import FortyTwo from './fortyTwo'; // PascalCase import/filename, camelCase export
```
Comment (ljharb) [inline README.md]: the comment says the import *and* the filename are PascalCase; so `FortyTwo` is correct

File: README.md [hunk 2]
```diff
@@ -2893,13 +2893,13 @@ Other Style Guides
...
+    import InsideDirectory from './insideDirectory'; // PascalCase import/filename, camelCase export
```
Comment (ljharb) [inline README.md]: same here

File: README.md [hunk 3]
```diff
@@ -2893,13 +2893,13 @@ Other Style Guides
...
 
     // bad
     import CheckBox from './check_box'; // PascalCase import/export, snake_case filename
-    import forty_two from './forty_two'; // snake_case import/filename, camelCase export
-    import inside_directory from './inside_directory'; // snake_case import, camelCase export
+    import forty_two from './fortyTwo'; // snake_case import/filename, camelCase export
```
Comment (ljharb) [inline README.md]: same here

File: README.md (hunk 1 above)
Comment (RslnMtvv) [reply README.md]: Thank you for explanation. I am really sorry about wasting your time!

Comment (ljharb) [reply README.md]: No problem. Thanks for the contribution!

**Pull request closed without merging**